addons/crm_iap_lead_enrich/models/iap_enrich_api.py

```diff
@@ -14,9 +14,10 @@ class IapEnrichAPI(models.AbstractModel):
     def _contact_iap(self, local_endpoint, params):
         account = self.env['iap.account'].get('reveal')
         dbuuid = self.env['ir.config_parameter'].sudo().get_param('database.uuid')
+        endpoint = self.env['ir.config_parameter'].sudo().get_param('reveal.endpoint', self._DEFAULT_ENDPOINT)
```

Comment:
Pro or Rar asked me to remove it, is it back ?

Reply:
removed it, I used it when testing myself.